### PR TITLE
Initial setup of locale files for ManageIQ::Providers::Azure

### PR DESCRIPTION
**Pull request by mzazrivec**
These files were created by a rake task:

    rake locale:plugin:find[ManageIQ::Providers::Azure]

run from ManageIQ git checkout.

### CODE REVIEW

File: locale/ManageIQ_Providers_Azure.pot [hunk 1]
```diff
@@ -0,0 +1,155 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the ManageIQ_Providers_Azure package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (Fryguy) [inline locale/ManageIQ_Providers_Azure.pot]: Should these be updated?

Comment (mzazrivec) [inline locale/ManageIQ_Providers_Azure.pot]: Hmm, don't know. Things will work just fine even without this info being filled in :smile: 

I'd say this is just a cosmetic issue, but I can definitely fix this if it's important.

Comment (djberg96) [inline locale/ManageIQ_Providers_Azure.pot]: I worry that if you don't update it now it will never get updated. :)

Comment (mzazrivec) [inline locale/ManageIQ_Providers_Azure.pot]: @djberg96 Alrighty, you convinced me :smile:

**Comment by djberg96** [comment]
👍

**Pull request merged**